hail_search/hail_search_query.py

```diff
@@ -695,7 +701,34 @@ def _sort_order(self, ht):
         return sort_expressions
 
     def _get_sort_expressions(self, ht, sort):
-        return self.SORTS[sort](ht)
+        sort_expressions = []
+        if sort in self.SORTS:
+            sort_expressions = self.SORTS[sort](ht)
+
+        elif sort in self.PREDICTION_FIELDS_CONFIG:
+            prediction_path = self.PREDICTION_FIELDS_CONFIG[sort]
+            sort_expressions = [-hl.float64(ht[prediction_path.source][prediction_path.field])]
+
+        elif sort == 'in_omim':
+            sort_expressions = self._omim_sort(ht, hl.set(set(self._sort_metadata)))
+
+        elif self._sort_metadata:
+            sort_expressions = self._gene_rank_sort(ht, hl.dict(self._sort_metadata))
+
+        else:
+            sort_field = next((field for field, config in self.POPULATIONS.items() if config.get('sort') == sort), None)
```

Comment:
maybe personal preference but I think just making this a list and accessing the first element with `[0]` is easier to read.  We expect `(field for field, config in self.POPULATIONS.items() if config.get('sort') == sort)` to have at most one element right?

Reply:
I prefer `next` because it stops iteration when it finds the match

Reply:
what about statically defining the `{config['sort']: field}` mapping so the lookup is `O(1)`?

Reply:
I had that originally, and you asked to change it so it would be part of the main `POPULATIONS` configs

Reply:
theres are max 5 populations, I think we are overthinking the optimization here

Reply:
Sure.. I think I was originally looking for something like:

```
POPULATIONS = {
   ...
 }
POPULATION_SORTS = {config['sort'] :field for field, config in POPULATIONS if config.get('sort')}
```
I agree that the optimization bit is unnecessary, so going to go ahead and approve!